docs/source/client_domains.rst

```diff
@@ -36,8 +36,10 @@ Pinning Requirements
 | aus5.mozilla.org | Firefox      | 42.0 and up | Nothing                                                       | None         | `Bug 1116409 <https://bugzilla.mozilla.org/show_bug.cgi?id=1116409>`_ | YES - No pinning requirements for some apps, and we can get certs for those that do pin. |
 +                  +--------------+             +                                                               +              +                                                                       +                                                                                          +
 |                  | Fennec       |             |                                                               |              |                                                                       |                                                                                          |
-+                  +--------------+             +---------------------------------------------------------------+              +                                                                       +                                                                                          +
-|                  | GMP          |             | "CN=DigiCert SHA2 Secure Server CA,O=DigiCert Inc,C=US" |br|  |              |                                                                       |                                                                                          |
++                  +--------------+-------------+---------------------------------------------------------------+              +-----------------------------------------------------------------------+                                                                                          +
+|                  | GMP          | 101.0 and up| Nothing                                                       |              | `Bug 1760527 <https://bugzilla.mozilla.org/show_bug.cgi?id=1760527>`_ |                                                                                          |
```

Comment:
This assumes bug 1760527 rolls out to release in 101 as currently planned.  We'll want to update the doc if/when the change goes to ESR as well.